pkg/ipamfetcher/customer.go

```diff
@@ -18,9 +18,17 @@ type customersResponse struct {
 }
 
 type customer struct {
+	Contacts     []Contact    `json:"Contacts"`
```

Comment:
Task: can you update the gateway-outgoing egress response validation section to match these new fields?

Reply:
Ooooh good call.  Yes I can.

Reply:
I changed two things in response to your good suggestion:
1. changed the code (where we formerly had "Custom Fields") to match the gateway-outgoing.yaml which has "custom_fields"
2. added "Contacts" array and "Contact" schema to the gateway-outgoing.yaml

(1) is safe.  Check here to confirm that both "Custom Fields" and "custom_fields" is always returned, and always identical:  https://ipam.net.atlassian.com/api/1.0/customers